triggers/daml/Daml/Trigger.daml

```diff
@@ -216,6 +232,7 @@ data TriggerAState = TriggerAState
   -- This will be used for dedupCreateCmd/dedupExerciseCmd helpers.
   , emittedCommands : [Commands]
   -- ^ Emitted commands in reverse because I can’t be bothered to implement a dlist.
+  , pendingContracts : Map CommandId [AnyContractId]
   , nextCommandId : Int
```

Comment:
Maybe add some haddocks for completeness?

Reply:
Good call, added.